### PR TITLE
Django 1.9

### PR DESCRIPTION
**Pull request by GregLeBarbar**
Features removed in 1.9 :
1. All models need to be defined inside an installed application or declare an explicit app_label. Furthermore, it isn’t possible to import them before their application is loaded. In particular, it isn’t possible to import models inside the root package of an application
2. The django.contrib.contenttypes.generic module is removed
3. django.db.models.loading is removed.

### CODE REVIEW

**Comment by safwanrahman** [comment]
Thanks @GregLeBarbar for the patch.
As ``django 1.8.x`` is still supported, we can not drop support for this version.
The [test failing](https://travis-ci.org/jazzband/django-authority/builds/147812681) seems related.

Can you please make the patch that it also support ``django 1.8.x`` so it will be backward compatible.

**Comment by GregLeBarbar** [comment]
Sorry for my late response
The tests seem to be ok now

File: authority/utils.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            app_path = sys.modules[app].__path__
+        except AttributeError:
+            continue
+        try:
+            imp.find_module('permissions', app_path)
+        except ImportError:
+            continue
+        __import__("%s.permissions" % app)
+        app_path = sys.modules["%s.permissions" % app]
+    LOADING = False
```
Comment (safwanrahman) [inline authority/utils.py]: A blank line is neccesary

**Comment by safwanrahman** [comment]
It looks cool. I think we also need to change in travis config to test it against ``django 1.9``. Can you please add it?
Otherwise I can look into that

File: authority/utils.py (hunk 1 above)
Comment (GregLeBarbar) [inline authority/utils.py]: The blank line was added in my last push !

**Comment by GregLeBarbar** [comment]
Yes, I modified the tox configuration for django 1.9

**Comment by safwanrahman** [comment]
Handing to @jezdez for a review

**Comment by sposs** [comment]
I'm using 1.9 and I really would like to use this lib... Thanks for providing the fixes.

**Comment by auvipy** [comment]
14b92a0dcd138f1bc9a6fac76040fb26b8bdbdf1 this adds 1.10 support

**Pull request closed without merging**